script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -371,17 +371,17 @@ So I'm the only one who's strange."
 "Yes, you're strange. There is no doubt about it."
 <Page103>
 I fought back the tears that were coming to my eyes.
-The same words hurt many times more than when the doctor said them, and I felt like I was going to break.
+These same words hurt many times more than when the doctor said them, and I felt like I was going to break.
 <Page104>
 But I knew it was coming.
-From the beginning, ---I've always known that I'm a child who shouldn't exist from the moment I woke up.
+From the beginning ---- from the moment I woke up from that incident, I knew that I was that child who shouldn't have been there.
```

Comment:
Is there a reason we went for `be there` over exist? `I knew that I was a child who shouldn't have existed` reads more fluently to me.
I think `a child` is more appropriate than `that child` also